reference-stores/next/components/Section/CtaStrip.js

```diff
@@ -0,0 +1,53 @@
+import Image from 'next/Image';
+import Link from 'next/Link';
```

Comment:
Casing:

```suggestion
import Link from 'next/link';
```

Reply:
Fixed in https://github.com/getnacelle/nacelle-js/pull/133/commits/9d3d0c95de5016f1095d9151bebaddaf07f9e863.